src/appleseed/renderer/modeling/camera/pinholecamera.cpp

```diff
@@ -416,6 +308,22 @@ DictionaryArray PinholeCameraFactory::get_input_metadata() const
             .insert("use", "optional")
             .insert("default", "-0.001"));
 
+    metadata.push_back(
```

Comment:
Maybe we should factorize perspective camera metadata like we already factorize common camera metadata (`CameraFactory::get_input_metadata()`).

Reply:
Done